js/mexc3.js

```diff
@@ -3638,186 +3645,169 @@ module.exports = class mexc3 extends Exchange {
          * @method
          * @name mexc3#fetchDeposits
          * @description fetch all deposits made to an account
-         * @param {string|undefined} code unified currency code
+         * @param {string} code unified currency code
          * @param {int|undefined} since the earliest time in ms to fetch deposits for
          * @param {int|undefined} limit the maximum number of deposits structures to retrieve
          * @param {object} params extra parameters specific to the mexc3 api endpoint
          * @returns {[object]} a list of [transaction structures]{@link https://docs.ccxt.com/en/latest/manual.html#transaction-structure}
          */
+        if (code === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchDeposits() requires a currency code argument');
+        }
         await this.loadMarkets ();
         const request = {
-            // 'currency': currency['id'] + network example: USDT-TRX,
-            // 'state': 'state',
-            // 'start_time': since, // default 1 day
-            // 'end_time': this.milliseconds (),
-            // 'page_num': 1,
-            // 'page_size': limit, // default 20, maximum 50
+            // 'coin': currency['id'] + network example: USDT-TRX,
+            // 'status': 'status',
+            // 'startTime': since, // default 90 days
+            // 'endTime': this.milliseconds (),
+            // 'limit': limit, // default 1000, maximum 1000
         };
         let currency = undefined;
-        if (code !== undefined) {
-            const rawNetwork = this.safeString (params, 'network');
-            params = this.omit (params, 'network');
-            if (rawNetwork === undefined) {
-                throw new ArgumentsRequired (this.id + ' fetchDeposits() requires a network parameter when the currency is specified');
-            }
-            // currently mexc does not have network names unified so for certain things we might need TRX or TRC-20
-            // due to that I'm applying the network parameter directly so the user can control it on its side
-            currency = this.currency (code);
-            request['currency'] = currency['id'] + '-' + rawNetwork;
+        const rawNetwork = this.safeString (params, 'network');
+        params = this.omit (params, 'network');
+        if (rawNetwork === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchDeposits() requires a network parameter when the currency is specified');
         }
+        // currently mexc does not have network names unified so for certain things we might need TRX or TRC-20
+        // due to that I'm applying the network parameter directly so the user can control it on its side
+        currency = this.currency (code);
+        request['currency'] = currency['id'] + '-' + rawNetwork;
```

Comment:
is it `currency` or coin here? 
![image](https://user-images.githubusercontent.com/43336371/194571332-f256ea27-039e-4e87-9a53-c69a1bcbe3ff.png)